code/game/objects/items/ego_weapons/non_abnormality/smiling.dm

```diff
@@ -0,0 +1,49 @@
+/obj/item/ego_weapon/city/smiling_pipe
+	name = "Long pipe"
+	desc = "A really long smoking pipe."
+	special = "Hitting someome will inflict smoke on them"
+	icon_state = "smiling_pipe"
+	force = "40"
+	attack_speed = 1.5
+	damtype = BLACK_DAMAGE
+	inhand_icon_state = "smiling_pipe"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	attack_verb_simple = list("Bashes", "Crushes")
+	attack_verb_continuous = list("Bashes", "Crushes")
+	var/hit_count = 0
+
+//Ok so basically if you hit someome with this they'll get covered in smoke
+/obj/item/ego_weapon/city/smiling_sword/attack(mob/living/target, mob/living/user)
+	if(!CanUseEgo(user))
+		return
```

Comment:
un-needed code. obj/item/ego_weapon already checks if the person can use EGO

Reply:
nevermind, apparently its standard practise to have this double-checked

Reply:
nevermind that nevermind, we have like 15 weapons shit-coded and that threw me off

Reply:
Brain fart